libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -221,6 +222,51 @@ class MapRouteLineTest {
         assertEquals(result[0], directionsRoute)
     }
 
+    @Test
+    fun retrieveDirectionsRoutesPrimaryRouteIsFirstInList() {
+        every { style.layers } returns listOf(primaryRouteLayer)
+        val primaryRoute: DirectionsRoute = getDirectionsRoute(true)
+        val altRoute: DirectionsRoute = getDirectionsRoute(false)
+        val directionsRoutes = mutableListOf(primaryRoute, altRoute)
+        val mrl = MapRouteLine(
```

Comment:
Not a big fan of abbreviations, could we rename this? Same thing happens to `altRoute` 😬